Dockerfile

```diff
@@ -40,6 +38,8 @@ ADD lerna.json /usr/src/
 
 COPY ./packages /usr/src/packages
 
+ADD yarn.lock /usr/src/
```

Comment:
`yarn.lock` changes quite frequently, so should be added to the image after other files.